community/suricata/APKBUILD

```diff
@@ -24,7 +24,7 @@ _enable_luajit=""
 _enable_rust=""
 
 case "$CARCH" in
-	armhf|armv7|x86_64)
+	aarch64|armhf|armv7|ppc64le|x86|x86_64)
```

Comment:
I'm pretty sure luajit isn't actually functional on at least aarch64.

Reply:
AFAIK it works on all but `s390x`

Reply:
Well, at least on my aarch64 machine it just instantly explodes

Reply:
`luajit` or `suricata`?
Btw. there's a PR to address `s390x`: #7666

Reply:
luajit